index.js

```diff
@@ -77,7 +77,7 @@ controller.hears(['register', 'signup', 'create account', 'sign up'], ['direct_m
 	}
 });
 
-controller.hears(['hi', 'hello', 'howdy', 'hallo', 'hoi'], ['direct_message', 'direct_mention'], function(bot, message) {
+controller.hears(['hi', 'hello', 'howdy', 'hallo', 'hoi', 'hey', 'aloha', 'whats up', 'oi', 'ey', 'yo'], ['direct_message', 'direct_mention'], function(bot, message) {
```

Comment:
Please add add `bonjour`, `namaste`, `hola`. Also, create a variable named `greetings` that contains these values.